PHP/Indentation Rules - heredoc end.tmPreferences

```diff
@@ -7,10 +7,10 @@
 	<string>source.php meta.heredoc-end.php</string>
 	<key>settings</key>
 	<dict>
-		<key>unIndentedLinePattern</key>
+		<!-- <key>unIndentedLinePattern</key>
```

Comment:
There should probably be a comment here explaining why this is commented-out.

Reply:
@wbond Added.